projects/media-viewer/src/lib/comments-summary/comments-summary.component.html

```diff
@@ -14,9 +14,9 @@ <h2>{{ title }}</h2>
           <tr *ngFor='let comment of comments; let i = index' [id]="'comment-row-'+i" class="govuk-table__cell">
             <td><span title="{{comment.createdByDetails.email}}">{{comment.createdByDetails.forename}} {{comment.createdByDetails.surname}}</span></td>
             <td>{{comment.createdDate | date : "hh:mm d MMMM yyyy" }}</td>
-            <td title="{{comment.commentText}}">
+            <td title="{{comment.content}}">
```

Comment:
Just a general question...
If this was to fix E2E tests, why are we changing the actual code?

Reply:
code was broken

Reply:
Linus changed the comment object. Not sure if this change is already in place in master. It's not that visible in master because it needs mock data to be shown anyway. In its current situation, mock data would show created date and author, but not the comment text.

Reply:
Mea culpa - intellij refactoring was on steroids.